src/freebox_api/aiofreepybox.py

```diff
@@ -118,6 +118,9 @@ async def open(self, host: str, port: str) -> None:
         cert_path = os.path.join(os.path.dirname(__file__), "freebox_certificates.pem")
         ssl_ctx = ssl.create_default_context()
         ssl_ctx.load_verify_locations(cafile=cert_path)
+        # Disable strict validating introduced in Python 3.13, which doesn't
+        # work with default Freebox certificates
+        ssl_ctx.verify_flags &= ~ssl.VERIFY_X509_STRICT
```

Comment:
```suggestion
        if ".fbxos.fr" in host or "mafreebox.freebox.fr" in host:
            # Disable strict validating introduced in Python 3.13, which doesn't
            # work with default Freebox certificates
            ssl_ctx.verify_flags &= ~ssl.VERIFY_X509_STRICT
```

Should we disable this SSL verification flag only on certain hosts ?

Reply:
Or as @olivierh65 is having an issue on this, should we completely disable SSL check for those hosts ?

Reply:
AftFurther investigation seems to confirm that the problem is not with the python version, but rather with certificates that are no longer valid.
Creating a new domain with a new let's encrypt certificate works fine.

Reply:
Still linked with the Python version as written above